include/nlohmann/json.hpp

```diff
@@ -2217,6 +2288,9 @@ class basic_json
     {
         assert_invariant();
         m_value.destroy(m_type);
+#if JSON_DIAGNOSTICS
+        m_parent = nullptr;
+#endif
```

Comment:
This should not be needed.

Reply:
Right - this was a leftover from debugging.